src/main/webapp/style.css

```
@@ -203,11 +190,16 @@ tr:nth-child(even) {
 .gridElement {
   display: grid;
   grid-template-columns: repeat(2, 50%);
-  grid-template-rows: auto auto auto auto auto;
+  grid-template-rows: auto auto 250px auto auto;
```

As discussed, I think the fourth column should be fixed due to the addition of subtitles.
```suggestion
  grid-template-rows: auto auto auto 250px auto;
```